### PR TITLE
[now dev] Add `host` property to the invoke payload

### PR DESCRIPTION
**Pull request by TooTallNate**
Fixes https://github.com/zeit/now-builders/issues/390.

### CODE REVIEW

File: src/commands/dev/lib/dev-server.ts [hunk 1]
```diff
@@ -548,6 +548,7 @@ export default class DevServer {
         const body = await rawBody(req);
         const payload: InvokePayload = {
           method: req.method || 'GET',
+          host: req.headers.host,
```
Comment (styfle) [inline src/commands/dev/lib/dev-server.ts:551]: I think this is already available in `getNowProxyHeaders()` below.

**Review by styfle — APPROVED** [review]

**Comment by styfle** [comment]
Does this need to be implemented in the backend too?

**Comment by TooTallNate** [comment]
No, production already sets this value. This is making it the same response shape for now dev as in production.

File: src/commands/dev/lib/dev-server.ts (hunk 1 above)
Comment (TooTallNate) [reply src/commands/dev/lib/dev-server.ts:551]: Same difference really, it's just a property lookup in either case.

**Pull request merged**